### DISCUSSION TITLE
Blocker to upgrade EDC to 0.7.2

### DISCUSSION THREAD

**Discussion by sabarishrl** [eclipse-tractusx/tractusx-edc#1383] [General]
We want to upgrade our EDC to 0.7.2. From the migration document and the readme, there are couple of new components included. One of them is IATP in the place of MIW. We want to know How can this be installed in our test environment, do we have any documentation around this? Or should we procure commercial license from SAP's DIM,

**Comment by syNack0** [comment]
@sabarishrl Hi, same problem here, any progress?

**Comment by lgblaumeiser** [comment]
Hello @sabarishrl @syNack0 , sorry, that I only saw this now. WIth the upcoming Jupiter Release of Catena-X, based on the release versions Eclipse Tractus-X 24/05 and 24/08, there is open source wallet anymore. The productive environment will most likely use the commercial DIM solution from SAP. Currently, there is no easy testsetup for a local data space, unfortunately.

There will be changes soon. One of the changes is, that the MIW team is working on a solution to provide the MIW again. The MXD team works with this MIW currently to setup a minimal viable dataspace again. This requires a current release candidate of the EDC for now to work. The development has not finished but I  expect the availability soon.

Another solution that will come soon is the availability of a wallet stub, that works as a replacement for the SAP DIM. It is not a complete wallet implementation but provides reasonable responses to all endpoints provided by the wallet. Unfortunately, this also is only available soon.

As a customer of Cofinity-X, you will receive access to a pre-prod environment of the upcoming productive Catena-X dataspace until end of August.

Concerning your original question, the IATP is only a protocol that is used to communicate between the provider EDC and the consumer wallet. And of course also to issue credentials. It is nowadays called Decentralized Claims Protocol (DCP).

**Comment by lgblaumeiser** [comment]
Closed as stale, please renew item if this is still an issue
